scripts/synthetix/facilitate_trade.js

```diff
@@ -50,6 +51,30 @@ const gasStationURL = {
   4: "https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/",
 }
 
+const toWEI = function (value, decimals) {
+  const h = Math.floor(decimals / 2)
+  const a = new BN(10).pow(new BN(h))
+  const b = new BN(10).pow(new BN(decimals - h))
+
+  return new BN(value * a).mul(new BN(b))
+}
+
+const BN = require("bn.js")
+// Could also use linkPrefix here.
+const estimationURLPrexix = {
+  1: "https://dex-price-estimator.gnosis.io//api/v1/",
+  4: "https://dex-price-estimator.rinkeby.gnosis.io//api/v1/",
```

Comment:
Is the double slash in the links intended? It looks strange.

Reply:
It sure does look strange.. but that is the link... Even if I remove the double slash it redirects me to the double slash.